src/components/Passports/index.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from "react";
+import PassportView from "./PassportView";
+import { Flex } from "rimble-ui";
+import {
+  Loading,
+  Container,
+  PassportCover,
+  PassportLabel,
+  PassportCountry,
+  PassportName
+} from "./styles";
+import {
+  getStoredValue,
+  storeValues,
+  eraseStoredValue
+} from "../../services/localStorage";
+
+class Passports extends Component {
+  constructor(props) {
+    super(props);
+    const { account, list } = props;
+    const passportId = getStoredValue("currentPassport", account);
+    const initialState = {
+      currentPassportId: passportId
+    };
+    if (passportId) {
+      initialState.currentPassport = list
+        ? list.find(passport => passport.id === passportId)
+        : null;
+    }
+    this.state = initialState;
+
+    // Bind methods to class
+    this.selectPassport = this.selectPassport.bind(this);
+    this.closePassport = this.closePassport.bind(this);
+  }
+
+  componentWillReceiveProps({ list }) {
```

Comment:
I'm not familiar with `componentWillReceiveProps`, but I read that this method (and others) [will be deprecated](https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops). Do you think we can have another approach here?

Reply:
Good call :)